service/service.go

```diff
@@ -542,7 +698,8 @@ func (s *service) doProbe(ctx context.Context) error {
 		}
 
 		if err := s.nodeProbe(ctx); err != nil {
-			return err
+			Log.Infof("nodeProbe failed: %s", err.Error())
+			// return err
```

Comment:
Remove commented line if not needed or add documentation here to describe why we aren't returning the error anymore

Reply:
done